src/components/Dialogs/DeleteFilesModal.tsx

```diff
@@ -21,7 +21,8 @@ const Title = ({ fileCount }: { fileCount: number }) => (
     <div>Delete Confirmation</div>
     <div className="flex gap-x-1">
       <div className="text-panel-text-important">{fileCount}</div>
-      &nbsp;Files
+      &nbsp;
```

Comment:
why do we have `&nbsp;` and not a padding? Isn't `&nbsp;` inconsistent between systems and fonts?

Reply:
`&nbsp;` is basically just a normal space. We write that because we can't put a "space" there. It cannot be padding because it has to follow the font/system's default word spacing